rust/processor/src/utils/counters.rs

```diff
@@ -202,7 +202,17 @@ pub static SINGLE_BATCH_DB_INSERTION_TIME_IN_SECS: Lazy<GaugeVec> = Lazy::new(||
     register_gauge_vec!(
         "indexer_processor_single_batch_db_insertion_time_in_secs",
         "Time taken to insert to DB for a single batch of transactions",
-        &["processor_name"]
+        &["processor_name", "task_index"]
+    )
+    .unwrap()
+});
+
+/// Count of transactions processed per batch
+pub static SINGLE_BATCH_NUM_TRANSACTIONS_PROCESSED_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
this seems like a redundant metric of `NUM_TRANSACTIONS_PROCESSED_COUNT`?

Reply:
good call!